packages/sit-onyx/src/components/OnyxTooltip/OnyxTooltip.vue

```diff
@@ -18,11 +20,15 @@ defineSlots<{
 
 const {
   elements: { trigger, tooltip },
-} = createTooltip({});
+  state: { isVisible },
+} = createTooltip({
+  trigger: computed(() => props.trigger),
+  debounce: 200,
+});
 </script>
 
 <template>
-  <div class="onyx-tooltip-wrapper">
+  <div class="onyx-tooltip-wrapper" :class="{ 'onyx-tooltip-wrapper--hidden': !isVisible }">
```

Comment:
Might be nicer to set the class on the tooltip element, where it is actually applied.